(doc) Update Verifier OS

pauby opened the pull request:
## Description Of Changes
Updated the Package Verifier operating system and specifications. Pulled it out to make it an include so we don't have to update it everywhere.

## Motivation and Context
It was wrong.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left-hand side)/
* [ ] Menu structure has been updated

## Related Issue
N/A

pauby commented:
@Windos can you confirm the details above are correct, from a technical standpoint?

Windos commented on input/shared/verifier-os.txt:
```diff
@@ -0,0 +1,5 @@
+We currently test on one operating system. More will be added in the future. The operating systems we test with:
+* Windows Server 2019
```
The list of inclusions above the base Server 2019 install is:

```
Windows Subsystem for Linux (WSL) enabled
.NET 3.5 and 4.8 preinstalled
Windows Updates installed through to 2023-01-12
```

The intention being that this covers the common dependencies that require a reboot on install, so not including them limits the ability to test dependant package.

pauby replied:
I've updated it. I didn't mention patching levels as that can change as we update the image. We don't want to keep updating it.

pauby commented:
I've updated this now and removed the shared file. If we need that in the future, we can add it. Otherwise, we should just refer directly to this page.

pauby commented:
I'm going to overhaul the doc completely, so I will close this and open a new PR.
